modules/cover_map/functions/CfgFunctions.hpp

```diff
@@ -1,6 +1,10 @@
-class CoverMap {
-	file = "modules\cover_map\functions\CoverMap";
-	class CM_bordersToMap {};
-	class CM_Init {};
-	class CM_Live {};
-};
\ No newline at end of file
+#include "..\script_component.hpp"
+class COMPONENT {
+	tag = COMPONENT;
+	class CoverMap {
+		file = "modules\cover_map\functions\CoverMap";
+		class bordersToMap {};
```

Comment:
Just for consistency what convention are we following for function names? Should they all start with capital letters or lower case letters? E.g. should it be `BordersToMap` or should they all be starting with a lower case letter?

Reply:
lowerCaseCamel

Reply:
of course for users it doesnt matter since the function calls are all case insensitive, but it is nice to have

Reply:
OK, there's no point going back in changing everything over to this format, I'll just try to follow this for all new functions.